### PR TITLE
SEQNG-1055 Use dataId in call to sequenceStart.

### PR DESCRIPTION
**Pull request by jluhrs**
Fixed bug where the notification send to the ODB to signal the start of the sequence execution was not using the dataId.

### CODE REVIEW

File: modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala [hunk 1]
```diff
@@ -568,4 +568,14 @@ object SeqTranslate {
     def toAction(kind: ActionType): Action[F] = fromF[F](kind, x.map(r => Result.OK(Response.Configured(r.sys.resource))))
   }
 
+  def dataIdFromConfig[F[_]: MonadError[?[_], Throwable]](config: Config): F[DataId] =
```
Comment (cquiroz) [inline modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala]: This is already defined at `ObserveActions`. see https://github.com/gemini-hlsw/ocs3/blob/develop/modules/seqexec/server/src/main/scala/seqexec/server/ObserveActions.scala#L110

Comment (jluhrs) [reply modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala]: Not exactly. The one in `ObserveActions` takes a `ObserveEnvironment[F]`

Comment (cquiroz) [reply modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala]: Could we merge them?

Comment (jluhrs) [reply modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala]: Done

**Pull request merged**